src/sinks/gcp/cloud_storage.rs

```diff
@@ -226,30 +224,19 @@ impl GcsSink {
         Ok(Box::new(sink))
     }
 
-    fn healthcheck(&mut self) -> crate::Result<Healthcheck> {
-        let builder = Request::builder()
-            .method(Method::HEAD)
-            .uri(self.base_url.parse::<Uri>()?);
+    async fn healthcheck(mut self) -> crate::Result<()> {
+        let uri = self.base_url.parse::<Uri>()?;
+        let mut request = http::Request::get(uri).body(Body::empty())?;
```

Comment:
This appears to have turned a HEAD request into a GET.

Reply:
My bad. Thanks!